src/main/kotlin/org/rust/ide/inspections/fixes/AttachFileToModuleFix.kt

```diff
@@ -164,16 +160,28 @@ private fun insertFileToModule(file: RsFile, mod: RsMod) {
     }
 
     WriteCommandAction.runWriteCommandAction(project) {
-        val child = mod.firstChild
-        val inserted = if (child == null) {
-            mod.add(modItem)
-        } else {
-            mod.addBefore(modItem, child)
-        } as RsModDeclItem
-        inserted.navigate(true)
+        insertModItem(modItem, mod).navigate(true)
     }
 }
 
+private fun insertModItem(item: RsModDeclItem, module: RsMod): RsModDeclItem {
+    val anchor = module.firstItem
+    val existingMod = generateSequence(anchor) {
+        it.getNextNonCommentSibling() as? RsElement
+    }
+        .filterIsInstance<RsModDeclItem>()
+        .lastOrNull()
```

Comment:
```suggestion
    val existingMod = anchor?.rightSiblings
        ?.filterIsInstance<RsModDeclItem>()
        ?.lastOrNull()
```

I suppose it does the same thing